src/app/busca-ativa/diabeticos/Diabetes.tsx

```
@@ -1,66 +1,87 @@
+		open: false,
+	});
+	const [filtrosAplicados, setFiltrosAplicados] = useState(false);
+	const [voltarGatilho, setVoltarGatilho] = useState(0);
+	const [tabelaData, setTabelaData] = useState([]);
+	const router = useRouter();
+	const path = usePathname();
+	const visao =
+		session?.user.perfis.includes(5) || session?.user.perfis.includes(8)
+			? "aps"
+			: "equipe";
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: path

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)